### ISSUE TITLE
Profiles dialog is broken in Python 3.11 NVDA

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#15650]
### Steps to reproduce:
1. Open NVDA
2. Press NVDA+ctrl+p to open the profile dialog
3. Try to navigate through the list or press tab and shift+tab to navigate the dialog.
4. Press escape to close the dialog

### Actual behavior:
Following error occurs and NVDA cannot navigate the dialog with tab and shift+tab. Also pressing escape does not close the dialog.
```
IO - inputCore.InputManager.executeGesture (23:08:44.597) - winInputHook (3060):
Input: kb(laptop):NVDA+control+p
DEBUG - gui.contextHelp.bindHelpEvent (23:08:44.621) - MainThread (2268):
Did context help binding for ProfilesDialog
DEBUG - gui.contextHelp.bindHelpEvent (23:08:44.625) - MainThread (2268):
Did context help binding for ListBox
ERROR - unhandled exception (23:08:44.625) - MainThread (2268):
Traceback (most recent call last):
  File "wx\core.pyc", line 3427, in <lambda>
  File "gui\blockAction.pyc", line 72, in funcWrapper
  File "gui\__init__.pyc", line 467, in onConfigProfilesCommand
  File "gui\configProfiles.pyc", line 58, in __init__
TypeError: Sizer.Add(): arguments did not match any overloaded call:
  overload 1: 'proportion' is not a valid keyword argument
  overload 2: argument 'proportion' has unexpected type 'float'
  overload 3: argument 1 has unexpected type 'ListBox'
  overload 4: argument 1 has unexpected type 'ListBox'
  overload 5: argument 1 has unexpected type 'ListBox'
  overload 6: argument 1 has unexpected type 'ListBox'
  overload 7: argument 1 has unexpected type 'ListBox'
  overload 8: argument 1 has unexpected type 'ListBox'
  overload 9: argument 1 has unexpected type 'ListBox'
```

### Expected behavior:
NVDA should handle the profile dialog as before.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-2966Z18522845 (2024.1.0.29662)

#### Windows version:
Windows 11 22 H2

#### Name and version of other software in use when reproducing the issue:
n/a

#### Other information about your system:
n/a

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes, NVDA 2023.3 Beta 4 works correctly

#### If NVDA add-ons are disabled, is your problem still occurring?
yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
